src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -8,6 +8,8 @@
     <DebugType>pdbonly</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <IsARM Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture.ToString().StartsWith(`arm`, System.StringComparison.InvariantCultureIgnoreCase))'">true</IsARM>
```

Comment:
@eerhardt do you know any less hacky way to tell if we are running on ARM?

I based my idea on your GH comment from 2017 ;p

https://github.com/Microsoft/msbuild/issues/539#issuecomment-289930591

Reply:
You are checking for if the build itself is happening on an ARM device, and not targetting an ARM device? If so, I'd say this is probably the best approach - using `RuntimeInformation`.  We do the same thing in corefx:

https://github.com/dotnet/corefx/blob/2a2d34df7db5d2617d12f55fe5ad8fd208117049/Directory.Build.props#L61-L63

One question - why `InvariantCultureIgnoreCase` instead of `OrdinalIgnoreCase`?

Reply:
@eerhardt thanks a lot!

> You are checking for if the build itself is happening on an ARM device, and not targetting an ARM device?

yes, exactly

> One question - why InvariantCultureIgnoreCase instead of OrdinalIgnoreCase?

for some reason, I have a habit to always use InvariantCultureIgnoreCase everywhere based on some bad experience, but I honestly don't remember what experience was it.

I just read more about `OrdinalIgnoreCase` and it looks like a better choice here. Thanks for the suggestion!